modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSConstants.java

```diff
@@ -312,10 +312,19 @@
                     " WHERE " + DESTINATION_IDENTIFIER + "=? AND " +
                     DURABLE_SUB_ID + "=?";
 
+    protected static final String PS_UPDATE_DURABLE_SUBSCRIPTION_BY_ID =
+            "UPDATE " + DURABLE_SUB_TABLE +
+            " SET " + DURABLE_SUB_DATA + "=? " +
+            " WHERE " + DURABLE_SUB_ID + "=?";
```

Comment:
Didn't do it since the then it'll be inconsistent with the rest of the strings defined in RDBMSConstants. Will add a separate PR refactoring the whole code in the class.